### PR TITLE
fix: make tests compatible with aiohttp 3.10

### PR DESCRIPTION
**Pull request by mweinelt**
Closes: #58

### CODE REVIEW

**Comment by Lash-L** [comment]
Thanks!

File: tests/test_parser.py [hunk 1]
```diff
@@ -3,5 +3,5 @@
 from anova_wifi import AnovaApi
 
 
-def test_can_create() -> None:
+async def test_can_create() -> None:
```
Comment (dotlambda) [inline tests/test_parser.py:6]:
```suggestion
@pytest.mark.asyncio
async def test_can_create() -> None:
```

Comment (mweinelt) [reply tests/test_parser.py:6]: Oh yeah, my bad.

Comment (Lash-L) [reply tests/test_parser.py:6]: I'm not sure if the asyncio mark decorator is still needed. I believe purest asyncio automatically sets asyncio_mode= auto and therefore the decorator is not needed. I could be mistaken. If it is still not working please put forth another PR

**Pull request merged**